packages/taquito/test/wallet/operation.spec.ts

```diff
@@ -1,12 +1,13 @@
+import { TestScheduler } from 'rxjs/testing';
 import { BlockResponse } from '@taquito/rpc';
-import { rxSandbox } from 'rx-sandbox';
 import { Context } from '../../src/context';
 import { WalletOperation } from '../../src/wallet';
 import { blockResponse } from './data';
 
-describe('WalletOperation', () => {
-  const toJSON = (x: any) => JSON.parse(JSON.stringify(x));
+describe('increasePaidStorageOperation', () => {
```

Comment:
from the file name doesn't seem to be a `increasePaidStorageOperation` specific test suite

Reply:
I think that might have been a copy/paste error. Will fix